_inc/client/main.jsx

```diff
@@ -275,6 +275,11 @@ class Main extends React.Component {
 		);
 	};
 
+	shouldShowSupportCard() {
+		// Do not show in settings page
+		return this.props.isSiteConnected && this.props.route.path !== '/settings';
```

Comment:
I think you would most likely want to hide it from all the Settings route, not just the main one, so it does not appear again when you switch to a different tab in settings (e.g. Jetpack > Settings > Writing).

Reply:
excellent

Reply:
@jeherve, addressed your feedback in 7b0c5a5b5